src/genie/libs/parser/iosxe/tests/test_show_ap.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+from unittest.mock import Mock
+
+from genie.metaparser.util.exceptions import SchemaEmptyParserError
+from genie.libs.parser.iosxe.show_ap import ShowApDot115GhzChannel
+
+
+# ==========================================
+# Unit test for 'show ap dot11 5ghz channel'
+# ==========================================
+class TestShowApDot115GhzChannel(unittest.TestCase):
+    """Unit test for 'show ap dot11 5ghz channel'"""
+
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+    golden_parsed_output1 = {
+    "channel_assignment": {
+        "chan_assn_mode": "AUTO",
+        "chan_upd_int": 12,
+        "anchor_time_hour": 7,
+        "channel_noise": "Enable",
```

Comment:
we respect the indentation and have the schema based on it. could you please update like below?
```
    "channel_update_contribution": {
        "noise": "Enable",
        "interference": "Enable",
        "load": "Disable",
        "device_aware": "Disable",
    }